.github/workflows/e2e-tests.yml

```
@@ -49,13 +46,50 @@ jobs:
+        echo "github.action_ref: ${{ github.action_ref }}"
+        echo "github.action_repository: ${{ github.action_repository }}"
+        echo "github.action_status: ${{ github.action_status }}"
+
     - name: Checkout e2e-tests repository
       if: inputs.calling-repo != 'goodmap-e2e-tests'
       uses: actions/checkout@v4
       with:
         path: ${{ inputs.e2e-tests-path }}
-        repository: problematy/goodmap-e2e-tests
-        ref: ${{ inputs.goodmap-e2e-version }}
+        repository: ${{ steps.determine-e2e-ref.outputs.repository }}
```

actionlint (1.7.8)

property "determine-e2e-ref" is not defined in object type {}

(expression)